var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -65,7 +65,8 @@ class Rocblas(CMakePackage):
         ('@4.0.0',  'ab44bf46b609b5a40053f310bef2ab7511f726ae'),
         ('@4.1.0',  'd175277084d3253401583aa030aba121e8875bfd'),
         ('@4.2.0',  '3438af228dc812768b20a068b0285122f327fa5b'),
-        ('@4.3.0',  '9cbabb07f81e932b9c98bf5ae48fbd7fcef615cf')
+        ('@4.3.0',  '9cbabb07f81e932b9c98bf5ae48fbd7fcef615cf'),
+        ('@4.3.1',  '9cbabb07f81e932b9c98bf5ae48fbd7fcef615cf')
```

Comment:
Hm, so tensile was updated in 4.3.1 but rocblas 4.3.1 doesn't use it? (I checked it, seems correct)